lib/kernel/src/application_controller.erl

```diff
@@ -528,19 +534,17 @@ check_conf_data([]) ->
 check_conf_data(ConfData) when is_list(ConfData) ->
     [Application | ConfDataRem] = ConfData,
     case Application of
-	{kernel, List} when is_list(List) ->
-	    case check_para_kernel(List) of
-		ok ->
-		    check_conf_data(ConfDataRem);
-		Error1 ->
-		    Error1
-	    end;
 	{AppName, List} when is_atom(AppName), is_list(List) ->
-	    case check_para(List, atom_to_list(AppName)) of
-		ok ->
-		    check_conf_data(ConfDataRem);
-		Error2 ->
-		    Error2
+	    case lists:keymember(AppName, 1, ConfDataRem) of
+		true ->
+		    ?LOG_WARNING("duplicate application config: ~ts~n", [AppName]);
```

Comment:
Sorry - I didn't think about this before, but since we know that this will be printed by the simple handler, maybe it would be better to create the string as you did with the erlang:display earlier (not using io_lib) ? I mean, since the simple handler also uses erlang:display, it cannot do the formatting, and thus prints the format string and the arguments separately. ?LOG_WARNING can take just a string as argument also.

Reply:
Pushed.

Reply:
Thanks!